plugins/rest/help.md

```diff
@@ -40,11 +40,11 @@ Example input:
   "authentication_type": "Basic Auth",
   "base_url": "https://httpbin.org",
   "basic_auth_credentials": {
-    "username": "user@example.com",
+    "username": "user@example.com" 
```

Comment:
Is that comma not needed here any more? 😄

Reply:
Oh it is, theres a few others as well thanks for pointing this out 😄